resources/lib/router.py

```diff
@@ -27,16 +27,15 @@ def router(argv):
     action = params.get('action')
 
     _kodi = kodiwrapper.KodiWrapper(addon_handle, addon_url, addon)
+    _tokenresolver = tokenresolver.TokenResolver(_kodi)
```

Comment:
So I wanted to avoid loading TokenResolver even in the cases where it wasn't needed.
But maybe that was overly aggressive and there's no real need to. I didn't profile it.